posthog/hogql_queries/utils/query_date_range.py

```diff
@@ -223,6 +223,9 @@ def use_start_of_interval(self):
         is_relative = delta_mapping is not None
         interval = self._interval
 
+        if self._date_range.explicitDate:
```

Comment:
cc: @jurajmajerik have you released hogql powered experiments yet? I think this is a bug there, where we still end up rounding when explicit dates are chosen. I'd expect the experiment results to be incorrect here 🤔  (or I'm missing some other place which adjusts this, haven't seen it though)

Reply:
Not yet, I'll add this to my list things to test!

Reply:
What's the bug here exactly? This will return False, i.e. no use of `toStartOf<Interval>` when `explicitDate` is set, which _is_ true for expeirments.

Reply:
sorry unclear, I meant without my change it's a bug, with my change its fine 🙈

Reply:
(and if you can confirm its fine indeed that's excellent 👍 )

Reply:
not a big deal tbh if we didn't realise it already ✅